sonar-ws/src/main/java/org/sonarqube/ws/client/WsClient.java

```diff
@@ -120,8 +160,12 @@
 
   /**
    * @since 6.3
+   * @deprecated since 7.0 use {@link #projectAnalyses()} instead
    */
-  ProjectAnalysisService projectAnalysis();
+  @Deprecated
+  ProjectAnalysisService projectAnalysisOld();
```

Comment:
![INFO](https://sonarsource.github.io/sonar-github/severity-info.png 'Severity: INFO') Do not forget to remove this deprecated code someday. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=squid%3AS1133)